### PR TITLE
Megaupdate

### PR DESCRIPTION
**Pull request by piousp**

### CODE REVIEW

File: mods/botaniacombat.pw.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 name = "BotaniaCombat"
-filename = "botaniacombat-1.20.1-0.9.1.jar"
+filename = "botaniacombat-1.20.1-1.1.2.jar"
```
Comment (piousp) [inline mods/botaniacombat.pw.toml:2]: Should be fine

File: mods/ct-overhaul-village.pw.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 name = "ChoiceTheorem's Overhauled Village"
-filename = "ctov-3.3.6.jar"
+filename = "ctov-3.4.2.jar"
```
Comment (piousp) [inline mods/ct-overhaul-village.pw.toml:2]: Villages, can't affect already generated structures. Fine

File: mods/guarding.pw.toml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 name = "Guarding"
-filename = "guarding-2.3.3.jar"
+filename = "guarding-2.4.0.jar"
```
Comment (piousp) [inline mods/guarding.pw.toml:2]: Client mod for improving shield handling

File: mods/hybrid-aquatic.pw.toml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 name = "Hybrid Aquatic"
-filename = "hybrid-aquatic-1.1.0.jar"
+filename = "hybrid-aquatic-1.2.2.jar"
```
Comment (piousp) [inline mods/hybrid-aquatic.pw.toml:2]: Sea creatures

File: mods/moonfabric.pw.toml [hunk 5]
```diff
@@ -1,13 +1,13 @@
 name = "moonfabric"
-filename = "moonfabric-1.0.0.jar"
+filename = "moonfabric-1.6.5.jar"
```
Comment (piousp) [inline mods/moonfabric.pw.toml:2]: Amulets and other loot

**Review by pedorich-n — APPROVED** [review]

**Pull request merged**